examples/rendering/render_grid.dart

```diff
@@ -0,0 +1,25 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math' as math;
+
+import 'package:sky/rendering.dart';
+import 'package:sky/theme/colors.dart' as colors;
+import 'solid_color_box.dart';
+
+Color randomColor() {
+  final List<Color> allColors = [
+    colors.Blue,
+    colors.Indigo
+  ].map((p) => p.values).fold([], (a, b) => a..addAll(b));
```

Comment:
It walks the list of color maps (int -> Color) grabs their values (List<Color>) and then does a fold over that list of lists to make one big list of Colors.  We use ..addAll instead of .addAll to make sure the fold function returns the resulting folded list.